eng/pipelines/templates/Apex_Tests_On_Windows_Daily.yml

```diff
@@ -217,3 +217,18 @@ stages:
             definition: $(resources.pipeline.ComponentBuildUnderTest.pipelineID)
             artifactName: "VS15"
             downloadPath: "$(Pipeline.Workspace)/artifacts"
+
+        - powershell: |
+            $zipPath = "$(Pipeline.Workspace)/artifacts/EndToEnd.zip"
+            $dest = "$(System.DefaultWorkingDirectory)/artifacts/EndToEnd/"
+            Write-Output "Extracting '$zipPath' to '$dest'"
+            Expand-Archive -Path "$zipPath" -DestinationPath "$dest"
+            $nugetExePath = "$(Pipeline.Workspace)/artifacts/NuGet.exe"
+            Write-Output "Copying '$nugetExePath' to '$dest'"
+            Copy-Item -Path "$nugetExePath" -Destination "$dest"
+          displayName: "Extract EndToEnd.zip"
+
+        - task: PowerShell@1
```

Comment:
Can we just call the SetupFunctionalTests.ps1 that's part of the repo?

I can't think of why we'd need the script coming from the end to end test zip.

Reply:
Add Extract EndToEnd.zip step due to SetupFunctionalTests.ps1 is included in EndToEnd.zip.

Reply:
> Can we just call the SetupFunctionalTests.ps1 that's part of the repo?

By default, DartLab doesn't clone the repo on test machines.